Common/Securities/SecurityCache.cs

```diff
@@ -294,6 +320,11 @@ public T GetData<T>()
         /// </summary>
         public IEnumerable<T> GetAll<T>()
         {
+            if (typeof(T) == typeof(Tick))
+            {
+                return _lastTickTrades.Concat(_lastTickQuotes).Cast<T>();
+            }
+
             IReadOnlyList<BaseData> list;
```

Comment:
Believe we still need to modify the other methods in the security cache for the behavior to be consistent, `Reset`, `TryGetValue`, `ShareTypeCacheInstance`. Think `HasData/GetData` should also use the new collections so we can avoid duplicating the Tick storage with `_dataByType`